os/booting-with-pxe.md

```diff
@@ -179,11 +179,10 @@ usr/share/oem/example.ign
 ```
 
 Add the `oem.cpio.gz` to your PXE boot directory and [append it][append-initrd] to the `initrd` line in your `pxelinux.cfg` and specify the path to the config:
+
```

Comment:
the previous lack of new line is was causing the formatting error. We can choose to keep the ellipses or not, what do you prefer?